index.html

```diff
@@ -60,12 +60,21 @@ <h2>Welcome to ROS Index</h2>
   </div>
  <hr>
   <div class="row">
-  	<h2>Release announcements</h2>
+  	<h2><a href="http://docs.ros.org/en/rolling/Releases.html">Active Releases</a></h2>
```

Comment:
Do we really want to point this to http://docs.ros.org/en/rolling/Releases.html ?  Since this is covering both ROS 1 and ROS 2, that might be confusing.

Reply:
I delinked the header.